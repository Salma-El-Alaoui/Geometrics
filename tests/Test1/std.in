C c1 1 1 1
C c2 2 2 2
C c3 3 3 3
C c2 1 2 3
C c4 a 1 1
C c4 -1 -1 1
C c5 1 1 -1
C c6 1
C c6 1 1 1 1 1 1
C 123 1 1 1
EXIT
