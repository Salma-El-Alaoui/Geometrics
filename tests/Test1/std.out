OK
#Element ajoute: c1
OK
#Element ajoute: c2
OK
#Element ajoute: c3
ERR
#Un element geometrique nomme c2 existe deja
ERR
#Parametres invalides
OK
#Element ajoute: c4
ERR
#Parametres invalides (Rayon negatif)
ERR
#Parametres invalides
ERR
#Parametres invalides
ERR
#Parametres invalides
